sdk/src/main/java/com/silanis/esl/sdk/examples/LocalLanguageExample.java

```diff
@@ -0,0 +1,39 @@
+package com.silanis.esl.sdk.examples;
+
+import static com.silanis.esl.sdk.builder.DocumentBuilder.newDocumentWithName;
+import static com.silanis.esl.sdk.builder.PackageBuilder.newPackageNamed;
+import static com.silanis.esl.sdk.builder.SignatureBuilder.signatureFor;
+import static com.silanis.esl.sdk.builder.SignerBuilder.newSignerWithEmail;
+
+import com.silanis.esl.sdk.DocumentPackage;
+import com.silanis.esl.sdk.DocumentType;
+
+/**
+ * This example shows how to send a package with signer local language.
+ *
+ * Available localLanguages: en, fr, de, it, ru, zh-CN, zh-TW, es, pt, nl, da, el, ja, ko, local
+ */
```

Comment:
What does this mean? Does it mean you can use the above languages as a local language?

Reply:
This was the old comment from the solution which is not valid any more. I've removed it now.